.gitlab-ci.yml

```diff
@@ -12,6 +11,7 @@ variables:
   GIT_STRATEGY:                    fetch
   GIT_DEPTH:                       100
   CARGO_INCREMENTAL:               0
```

Comment:
this can be removed if it's working another way with sccache now

Reply:
Turning incremental compilation creates a bit of a runtime overhead and bloats `target/` which is often cached in Rust CI setups, hence why I believe it's enabled here.

Moreoever, sccache cannot really cache incrementally compiled Rust and since we use a Redis cache in our CI runs, I think we should leave that env var as it is, disabling the incr. comp, to speed up our CI runs.

Reply:
cool, good to know. A while ago I did some blackbox testing and it showed me that sccache worked better with `CARGO_INCREMENTAL=0`